frontend/components/Form/view.ts

```diff
@@ -315,20 +314,20 @@ export default class FormView extends View {
   changeFloatBtn(showClass: string): void {
     if (showClass === FORM_CLASS.REMOVE_BTN) {
       this.$remove.classList.remove(CLASS.HIDDEN)
-      this.$clear.classList.add(CLASS.HIDDEN)
     } else if (showClass === FORM_CLASS.CLEAR_BTN) {
       this.$remove.classList.add(CLASS.HIDDEN)
-      this.$clear.classList.remove(CLASS.HIDDEN)
     }
   }
 
-  clearForm() {
+  clear() {
     this.$amount.value = ''
-    this.$date.value = ''
+    this.$date.value = formatDate(new Date())
     this.$item.value = ''
     this.$category.value = ''
     this.$payment.value = ''
     this.invoiceId = 0
+    this.setCategoryType(CONSTANT.EARNING)
```

Comment:
```suggestion
    this.setCategoryType(CONSTANT.SPENDING)
```
초기 로딩시 지출이 선택되어야 합니당!